custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -28,6 +31,31 @@ class CloudIssuePage(
         }
     }
 
+    override fun editDescription(description: String): CloudIssuePage {
+        driver
+            .findElement(By.cssSelector("div[data-test-id = 'issue.views.field.rich-text.description']"))
+            .click();
+
+        val descriptionForm = driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.cssSelector("div[data-test-id='issue.views.field.rich-text.editor-container']"))
+            )
+
+        Actions(driver)
+            .sendKeys(description)
+            .perform()
+
+        descriptionForm
+            .findElement(By.cssSelector("button[data-testid='comment-save-button']"))
```

Comment:
Does it work? I'd expect this:
```suggestion
            .findElement(By.cssSelector("[data-test-id='comment-save-button']"))
```

Reply:
`Save` button has the attribute without the dash but i'll remove `button` tag from the selector

Reply:
Whooooah, nice find!